torch/csrc/jit/test_jit.cpp

```diff
@@ -912,7 +923,69 @@ void testControlFlow() {
   REQUIRE(3 == run_binary("if_test", 3, 2));
   REQUIRE(2 == run_binary("if_one", 2, 3));
   REQUIRE(2 == run_binary("if_one", 3, 2));
-  REQUIRE(256 == run_binary("while_test",2,0));
+  REQUIRE(256 == run_binary("while_test", 2, 0));
+}
+
+const static auto if_test_expected = R"JIT(  %2 = aten::lt(a, b)
+  %3 = prim::TensorToNum(%2)
+  if %3:
+    c = b
+  else:
+    c = a
+  return c
+)JIT";
+
+const static auto if_one_expected = R"JIT(  %2 = aten::lt(a, b)
```

Comment:
Can you use the expect tests capability from Python instead of doing these tests as hardcoded literals? This will make it a lot easier to update the tests if pretty printing changes, as you will be able to just `--accept` the changes.